docs/reference/api-v1alpha.md

```diff
@@ -916,4 +916,546 @@ curl -i \
+    "updated_by": "02984c87-efe8-4ea1-bcac-9511a34a3df3",
+    "updated_at": "2023-06-07T09:24:04.000000Z",
+    "subject_rules": [
+      {
+        "type": "ANY",
+        "subject_id": ""
+      }
+    ],
+    "state_message": "",
+    "state": "USABLE",
+    "project_id": "a426b4db-1919-483d-926d-06ba1320b209",
+    "organization_id": "7304b7f9-7482-46d4-9b95-3cd5a6ef2e6f",
+    "object_rules": [
+      {
+        "type": "BRANCH",
+        "pattern": "",
+        "match_mode": "ALL"
+      },
+      {
+        "type": "TAG",
+        "pattern": "",
+        "match_mode": "ALL"
+      },
+      {
+        "type": "PR",
+        "pattern": "",
+        "match_mode": "ALL"
+      }
+    ],
+    "name": "DTName123",
+    "last_deployment": null,
+    "id": "e79d3d1c-61cc-4e07-ba32-f86694f5e80d",
+    "description": "DT description",
+    "created_by": "02984c87-efe8-4ea1-bcac-9511a34a3df3",
+    "created_at": "2023-06-01T10:07:36.000000Z",
+    "bookmark_parameter3": "",
+    "bookmark_parameter2": "",
+    "bookmark_parameter1": "",
+    "active": true
+  }
+]
+```
+
+**Example**
+
+```
+curl -i -H "Authorization: Token {api_token}" \
+     "https://{org_name}.semaphoreci.com/api/v1alpha/deployment_targets?project_id=:project_id&target_name=:target_name"
+```
+
+### Creating a target
+
+This API endpoint allows you to create a new deployment target and assign it to a specific project.
+
+```
+POST {org_name}.semaphoreci.com/api/v1alpha/deployment_targets?project_id=:project_id
+```
+
+**Request Body**
+
+The request body should be a JSON object, encapsulating details about the deployment target to be created. The available fields are as follows:
+
+  - `name` (**required**) - Unique name for the target within the project.
+  - `project_id` (**required**) - UUID of the project.
+  - `unique_token` (**required**) - Idempotency UUID token.
+  - `description` (*optional*) - A description of the target.
+  - `url` (*optional*) - The URL of the target.
+  - `bookmark_parameter1`, `bookmark_parameter2`, `bookmark_parameter3` (*optional*) - Bookmark parameters - string values that represent the names of the promotion parameters. You can later use values of those parameters to [filter deployments in deployment history](/reference/deployment-targets-yaml-reference/#bookmark_parameter1-bookmark_parameter2-bookmark_parameter3).
+  - `subject_rules` (*optional*) - A list of subject rules structured as `{"type": RULE_TYPE, "subject_id": ID}`. The `RULE_TYPE` can be one of `"ANY"`, `"USER"`, `"ROLE"`, `"AUTO"` and the `ID` should be the name of the role (which must be valid for the project) or UUID of a user. For `USER` type you can use `{"type": "USER", "git_login": GITLOGIN}` where `GITLOGIN` is a user's git handle. The user must be assigned to the project to be used in the rule. The promotion is automatically triggered for the `"AUTO"` rule, provided it is allowed. In the case of `ANY` and `AUTO` rules, there is no need to specify the `subject_id`.
```

Comment:
Let's also add a bit of explanation at the begging of the line for what are the subject rules actually. Something like:

> - `subject_rules` (*optional*) - Configures **who** can trigger a promotion of the given deployment target. It should be a  list of elements structured as...